packages/Adapters/POD_C/src/DTK_C_API.h

```diff
@@ -0,0 +1,49 @@
+#ifndef DTK_NONAME_H
+#define DTK_NONAME_H
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// QUESTION: Do we want to be able to track missed entities?
```

Comment:
At some point we do but the point cloud operators don't have this capability yet. We may want to add that as part of this work.